warehouse/admin/static/js/warehouse.js

```diff
@@ -185,3 +185,42 @@ if (malwareReportsTable.length) {
   new $.fn.dataTable.Buttons(table, {buttons: ["copy", "csv", "colvis"]});
   table.buttons().container().appendTo($(".col-md-6:eq(0)", table.table().container()));
 }
+
+// Link Checking
+var links = document.querySelectorAll("a");
+links.forEach(function(link){
+  if (link.dataset.checkLinkUrl !== undefined) {
```

Comment:
Optimize: I _think_ you could replace this selector with `"a[data-check-link-url]"` and thus not iterate over every link on the page.

Since admin uses jQuery, we could use it with `var links = $('a[data-check-link-url]')` and then `link.each(function...` - but I don't mind native JS.

Reply:
I'll stick with plain js, but good note.

Reply:
eb28ba47c